azext_iot/core/help.py

```diff
@@ -0,0 +1,89 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+"""
+Help updates for CLI core commands.
+"""
+
+from knack.help_files import helps
+
+
+# TODO - CMS Preview - help additions to core
+def patch_core_help():
+
+    # add Hub create examples for ADR properties
+    if "iot hub create" in helps:
```

Comment:
This is because of the desire to patch over cli core help?

Reply:
Yeah, this is an artifact of separating core / extension help for CMS features depending where the core help modules reside - if they exist in core, this will append to them.

However, they live in this extension for now, so we could combine them